ui/src/pages/Alert/Shield/ShieldObjInput.tsx

```diff
@@ -145,28 +149,32 @@ export default function ShieldObjInput({
     return (
       <Row gutter={8}>
         <Col span={8}>
-          <Form.Item
-            rules={[
-              {
-                required: true,
-                message: intl.formatMessage({
-                  id: 'src.pages.Alert.Shield.639D1A8C',
-                  defaultMessage: '请选择',
-                }),
-              },
-            ]}
-            name={clusterFormName}
-          >
-            <Select
-              mode="multiple"
-              maxCount={1}
-              allowClear
-              options={getOptionsFromType(clusterFormName[1])}
-              placeholder={intl.formatMessage({
-                id: 'src.pages.Alert.Shield.B0133BD9',
-                defaultMessage: '请选择集群',
-              })}
-            />
+          <Form.Item noStyle dependencies={[nextFormName]}>
+            {() => (
```

Comment:
Why use `{() => <Component />}` instead of `<Component />` here?

Reply:
If you use `<Component />` directly, it will not be rendered. Maybe in `antd`'s view, these two ways of writing are different.
![image](https://github.com/oceanbase/ob-operator/assets/62841398/c5de49ad-a5b4-4ab3-99ea-0e2569164d72)